### ISSUE TITLE
minimal.rb is crashing when downloading stylesheets.zip

### ISSUE THREAD

**Issue by meleu** [lewagon/rails-templates#154]
## TL;DR

The `minimal.rb`, [in line 25](https://github.com/lewagon/rails-templates/blob/master/minimal.rb#L25), tries to download `https://github.com/lewagon/rails-stylesheets/archive/more-js.zip`, but in the [rails-stylesheets repo](https://github.com/lewagon/rails-stylesheets/branches), the `more-js` branch doesn't exist.

If we replace every mention of `more-js` with `master`, the template works.

---

## Long version

In the AirBnB day 1 the students are instructed to start a new Rails project using the `minimal.rb` template:
```
rails new \
  -d postgresql \
  -m https://raw.githubusercontent.com/lewagon/rails-templates/master/minimal.rb \
  CHANGE_THIS_TO_YOUR_RAILS_APP_NAME
```

When I do it on my end the command crashes with this error:
```
         run    curl -L https://github.com/lewagon/rails-stylesheets/archive/more-js.zip > stylesheets.zip from "."
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100    14  100    14    0     0     10      0  0:00:01  0:00:01 --:--:--    19
         run    unzip stylesheets.zip -d app/assets && rm -f stylesheets.zip && rm -f app/assets/rails-stylesheets-more-js/README.md from "."
Archive:  stylesheets.zip
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.
unzip:  cannot find zipfile directory in one of stylesheets.zip or
        stylesheets.zip.zip, and cannot find stylesheets.zip.ZIP, period.
```

This is happening because the contents of `stylesheets.zip` is a plaintext with a 404 error message. And the 404 message happens because the `more-js` branch doesn't exist anymore.

I tried editing the `minimal.rb`, changing all mentions to `more-js` with `master`, and it apparently works.
